### DISCUSSION TITLE
rootless podman with ldap user

### DISCUSSION THREAD

**Discussion by allangdtl** [containers/podman#24466] [Q&A]
Currently, we have setup an Ubuntu 24.04 server and installed podman on it. The local user had no issues as it is rootful as well. However, we have several ldap users connected to the server and I set up one of them according to: 
https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md

However, when trying to build, it fails with:
`sd-bus call: Interactive authentication required.: Permission denied
: exit status 1
ERRO[0002] did not get container create message from subprocess: EOF`

I tried doing 'podman system migrate' and it did not help.

**Comment by eriksjolund** [comment]
Quote from
[man subuid(5)](https://man7.org/linux/man-pages/man5/subuid.5.html)

_"The delegation of the subordinate uids can be configured via the subid field in /etc/nsswitch.conf file"._


Maybe you need to configure the _subid_ field in the file _/etc/nsswitch.conf_?


(just a wild guess)
